config/locales/cy/cy.yml

```diff
@@ -984,6 +985,7 @@ cy:
               blank: Mae'n rhaid i chi roi gwybodaeth ar gyfer y maes hwn
             address_building:
               blank: Rhowch enw neu rif adeilad cyfeiriad yr atebydd
+            blank_ccd_address: :'activemodel.errors.models.respondent.attributes.address_building.blank'
```

Comment:
I cannot see anywhere this is being used ?  The correct message is invalid_ccd_address ?